tools/http_server.py

```diff
@@ -178,7 +178,7 @@ def spawn():
     while any(not s.thread.is_alive() for s in servers):
         sleep(0.01)
     try:
-        print("ready")
+        print "ready"
```

Comment:
😳

Reply:
so it doesn't print parens when executed in 2.7

Reply:
Not true:
```
% python2 -c 'print("hello")' 
hello
```
Anyways, no big deal. We are ways away from python3! Once v8 builds with it then we can care.